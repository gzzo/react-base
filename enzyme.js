import Enzyme from 'enzyme'

// see https://github.com/enzymejs/enzyme/issues/2429
import Adapter from '@wojtekmaj/enzyme-adapter-react-17'

Enzyme.configure({ adapter: new Adapter() })
