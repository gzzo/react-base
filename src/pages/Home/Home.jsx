import React from 'react';

import { Page } from 'components/Page/Page';


export class Home extends React.Component {
  render() {
    return (
      <Page>
        Homepage
      </Page>
    );
  }
}
