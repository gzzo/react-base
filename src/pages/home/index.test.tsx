import React from 'react'
import { shallow } from 'enzyme'

import Home from './index'

describe('pages/home', () => {
  test('matches snapshot', () => {
    const wrapper = shallow(<Home />)
    expect(wrapper).toMatchSnapshot()
  })
})
