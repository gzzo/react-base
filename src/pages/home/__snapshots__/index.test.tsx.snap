// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pages/home matches snapshot 1`] = `
<Page>
  Home
</Page>
`;
